packages/exhibitor/build

```diff
@@ -10,6 +10,9 @@ rm -rf "$PKG_PATH/usr/zookeeper/"{src,docs,contrib}
 mkdir -p "$PKG_PATH/bin"
 ln -s "$PKG_PATH/usr/zookeeper/bin/zkCli.sh" "$PKG_PATH/bin/zkCli.sh"
 
+# Explicitly set JAVA_HOME to our java package
+export JAVA_HOME="$(find /opt/mesosphere/packages -type d -name java--*)/usr/java"
```

Comment:
I noticed that this is not on master, and I suspect we want it there as well... @BenWhitehead wdyt?